packages/components/Callout/package.json

```diff
@@ -39,12 +39,24 @@
   },
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
-    "@types/react-native": "^0.66.0",
     "@fluentui-react-native/scripts": "^0.1.1",
-    "react-native": "^0.66.0"
+    "@types/react-native": "^0.68.0",
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
   },
   "peerDependencies": {
-    "react": ">=17.0.2",
-    "react-native": ">=0.66.0"
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
```

Comment:
For "react" should we use ">=17.0.2" or "17.0.2"? And for "react-native" should it be ">=0.68.0" or "^0.68.0"?

This question applies to multiple packages. `yarn rnx-dep-check` changed it to what you see above.

Reply:
I think it should be "17.0.2", React Native seems to want a specific version of React down to the patch. 
I've seen "^" more than ">=", and I think our use of the latter may have caused some bugs with publishing at times?
In general I trust the output of `rnx-dep-check` more than a convention we've had in our repo, since it's used across multiple repos to manage dependencies

Reply:
Alright, thanks. I'll keep it as-is then.